modules/fishhook/1.0/scr/R/run_fishhook.R

```diff
@@ -0,0 +1,64 @@
+### Run R package dNdS to determine the significantly mutated genes ###
+
+#!/usr/bin/env Rscript
+#
+# Notes:
+#   This script is intended for use with the snakefile for fishHook in LCR-modules.
+#   It uses the snakemake objects to find maf file.
+#   Additionaly, it can support the genes of targeted panel to restrict SMG search.
+#
+
+# Load packages -----------------------------------------------------------
+message("Loading packages...")
+suppressWarnings(
+  suppressPackageStartupMessages({
+    library(fishHook)
+    library(gTrack)
+    library(rtracklayer)
+  })
+)
+#load maf and convert to GRange
+maf = dt2gr(fread(snakemake@input[[2]]))
+
+#split maf to numerous tiles
+tiles = gr.tile(seqinfo(maf), snakemake@params[[1]])
+
+events = maf %Q% (Variant_Classification != 'Silent')
+
+if(file.exists(snakemake@params[[2]])){
+  message("Running FishHook with Coveriate...")
+  fish_tiles = Fish(hypotheses = tiles, 
+                    events = events, 
+                    eligible = eligible, 
+                    idcol = 'Tumor_Sample_Barcode',
+                    use_local_mut_density=TRUE)
+  
+  coveriate_data = gr.sub(import(snakemake@params[[2]]), 'chr', "") ## import from bed then gUtils::gr.sub to strip 'chr' identifier
```

Comment:
Does this applies to any genome build? Or FishHook does not considers genome build at all? Just wonder if the `chr` should always be stripped or is dependent on the grch37/hg38

Reply:
I saw later in the CANGELOG that both grch37 and grch38 are supported - is it also important whether the maf `Chromosome` should be without chr prefix?

Reply:
FishHook does not consider genome build at all since it only use the maf file. however the covariate file does need to be the same type. But you are right, I don't think I need the gr.sub if both covariate and maf is the same type

Reply:
Done